std/typecons.d

```diff
@@ -1177,8 +1177,16 @@ if (distinctFieldNames!(Specs))
         size_t toHash() const nothrow @trusted
         {
             size_t h = 0;
-            foreach (i, T; Types)
-                h += typeid(T).getHash(cast(const void*)&field[i]);
+            static foreach (i, T; Types)
+            {{
+                const k = typeid(T).getHash(cast(const void*) &field[i]);
```

Comment:
Should use trusted escape here instead of on the whole function

Reply:
done